openapi.yaml

```diff
@@ -21975,12 +22697,21 @@ components:
                 type: array
                 items:
                   $ref: '#/components/schemas/HostnameGeneratorItem'
+                default: null
```

Comment:
I've not removed a couple of instances of these using overlay - because it does not cause anything to break.
@lobkovilya 
The other instance is in HostnameGeneratorItem - `spec.extension.config`